### DISCUSSION TITLE
Première annonce : Discussion Globale disponible !

### DISCUSSION THREAD

**Discussion by kilian-goetz** [Neighbook/.github#4] [Annonces]
@UTBM-FISA-TutoredProject/tutored-project

:loudspeaker:***Bonne nouvelle :*** Un chanel global nommé **discussion** sur l'organisation Github est désormais disponible ✔️ . Les sondages et les annonces se feront désormais ici de façon à garder une trace permanente. 

:newspaper: ***Informations :*** 
- Le sondage pour le nom du projet sera réalisé dans la catégorie en question (Fin des google form). 
- Des sondages par équipe seront réalisé pour désigner un "chef d'équipe" par équipe. **Son rôle ?** Former les membres de son équipe qui ne connaissent pas les technos utilisées selon l'équipe. De surcroît, il sera en charge de faire les codes review de son équipe ainsi que de participer aux réunions entre les différents référents de chaque équipe.
- Une [documentation générale](https://utbm-fisa-tutoredproject.github.io/Documentation/#/) du projet est désormais disponible. Celle-ci est générer grâce à Docsify et s'écrit en markdown. Vous trouverez dedans, le cahier des charges, des tutoriels pour les technologies, les règles à respecter pour le Github et bientôt une liste des personnes à contacter. Si vous souhaitez participer à l'élaboration de la documentation vous pouvez il suffit de me contacter pour que je vous octroie les droits.

**Comment by follyjohn** [comment]
Super @Beladric !
